main/main.cpp

```diff
@@ -3170,6 +3170,8 @@ Error Main::setup2(bool p_show_boot_logo) {
 		DisplayServer::set_early_window_clear_color_override(false);
 
 		GLOBAL_DEF_BASIC(PropertyInfo(Variant::STRING, "application/config/icon", PROPERTY_HINT_FILE, "*.png,*.bmp,*.hdr,*.jpg,*.jpeg,*.svg,*.tga,*.exr,*.webp"), String());
+		GLOBAL_DEF_BASIC(PropertyInfo(Variant::STRING, "application/config/icon_dark", PROPERTY_HINT_FILE, "*.png,*.bmp,*.hdr,*.jpg,*.jpeg,*.svg,*.tga,*.exr,*.webp"), String());
+		GLOBAL_DEF_BASIC(PropertyInfo(Variant::STRING, "application/config/icon_tinted", PROPERTY_HINT_FILE, "*.png,*.bmp,*.hdr,*.jpg,*.jpeg,*.svg,*.tga,*.exr,*.webp"), String());
```

Comment:
Since these are only used for iOS, I would suggest putting them in the iOS export preset instead. We can still have logic so that all the specific advanced cuts can be inferred from a "base icon" that would be a non-advanced setting in the export preset.

If other platforms start having support for this kind of dark/tinted icons, then we could refactor and put it in the project settings, but for now I'd keep it local (like we've done for Android in #97517 for example).

Reply:
Removed for project settings and added generic base icon to generate the rest, also move all except base 1024x1024 and App Store (have different requirements) to advance settings (it was too cluttered, and usually it's the same rescaled icon anyway).

<img width="845" alt="Screenshot 2024-10-04 at 12 26 50" src="https://github.com/user-attachments/assets/176cd87a-22e4-4776-bb7a-6e35b2f299d9">